.github/SECURITY.md

```diff
@@ -0,0 +1,122 @@
+# Oppia Vulnerability Disclosure Process
+
+This vulnerability disclosure process describes how we accept and respond to security vulnerabilities from both Oppia developers and others. Our process follows 4 steps: Report, Investigate, Remediate, and Disclose.
+
+## Report
+
+Reporters should email [security@oppia.org](mailto:security@oppia.org) with:
+
+* A description of the problem.
+* Steps we can follow to reproduce the problem.
+* Affected versions.
+* If known, mitigations for the problem.
+
+We will respond within 3 days of the reporter’s email to acknowledge receipt of their report. Here is a template acknowledgement email:
+
+```
+Hi $REPORTER,
+
+Thank you for reporting this problem to us. We are currently investigating and will reach out sometime in the next 7 days once we have decided how to move forward or if we have any questions.
+
+Thanks,
+
+$OPPIA_SECURITY_TEAM_MEMBER
+```
+
+## Investigate
+
+Immediately upon receiving a report of a security problem, a member of Oppia’s security team (the tech leads for now) will assemble a vulnerability response team (VRT). This team should:
+
+* Include an Oppia tech lead.
+* Include developers (most likely 1-2) with expertise in the part of the app affected by the problem.
+* Include as few developers as possible to avoid premature publication of the problem.
+
+The tech lead will designate one VRT member as the VRT lead responsible for driving our response. The VRT lead will immediately (ideally within 24 hours) investigate the report and classify it as:
+
+* Won’t fix: The app is working as intended, the bug is not important enough for us to spend resources fixing, or the requested feature is not something we plan to implement.
+* **Bug**: The problem identified is legitimate, but it is not a security issue. It will be resolved through our normal bug-fixing process.
+* **Feature request**: The report identifies a new feature that should be implemented. It will be handled through our normal feature-development process.
+* **Low-severity vulnerability**: The report identifies a security vulnerability that does not meet the high-severity criteria. It will be resolved through our normal bug-fixing process.
+* **High-severity vulnerability**: The report identifies an exploitable security vulnerability that, if exploited, could result in any of the following:
+  * (Confidentiality) Unauthorized access to personal user data.
+  * (Integrity) Unauthorized edits to any user data.
+  * (Availability) Degraded system performance of the platform for users.
```

Comment:
What does "user data" mean? Do models that are not directly tied to users (like config variables) count as "user data"?

Reply:
I lean towards classifying any unauthorized edits as a high-severity vulnerability. @vojtechjelinek @seanlip @BenHenning wdyt?

Reply:
Yeah I agree I would change "user data" to just "data".

Reply:
done